Sources/Purchasing/StoreKit1/ProductsFetcherSK1.swift

```diff
@@ -255,3 +255,9 @@ private extension ProductsFetcherSK1 {
 // @unchecked because:
 // - It has mutable state, but it's made thread-safe through `queue`.
 extension ProductsFetcherSK1: @unchecked Sendable {}
+
+#if hasFeature(RetroactiveAttribute)
+// Conformance should be safe since it is only used as dictionary key
```

Comment:
I recommend creating an `UncheckedSendable<T>` wrapper so you only ignore warnings in places where it might actually be safe rather than everywhere.